lib/index.js

```diff
@@ -24,7 +24,7 @@ export { default as VirtruLayout } from './components/layout/virtruLayout';
 export { default as TopMenuDropdown } from './components/dropdown/topMenuDropdown';
 // eslint-disable-next-line import/no-cycle
 export { default as TopMenuDropdownWrap } from './components/dropdown/topMenuDropdownWrap';
-
+// eslint-disable-next-line import/no-cycle
```

Comment:
This is a little concerning. What file introduced the cyclical dependency?

Reply:
I think this is because in this file we are exporting
Switch component and ItemOption component which internally also imports Switch. 
I don't think it's something bad.
So what do you say?

Reply:
Yes, I think you are right. `itemOptions` has `import { Switch } from '@';`. Why do we do that instead of use a relative import?

Reply:
But no matter jslint error, I think we should avoid "bad" looking relative import